### PR TITLE
Instruct users to move sites out of the testing org to go live

### LINKED ISSUE #940: New Doc Proposal: Moving a Beta Site Out of the Drupal 8 Testing Org to take it live
# Moving Drupal 8 Sites Created During Beta out of the Testing Organization
## Description
If we can determine that users who created sites during the Drupal 8 Beta intend on taking them live on Pantheon, we should document the migration and upgrade paths. 

## Outline
1. Caveats (beta-13 and previous have no upgrade path)
2. Export Site
3. Create new D8 site
4. Import DB, Files, Code (large site import doc)

## Expected Audience
Users who created Drupal 8 beta sites that are up-to-date running beta 14 or later. 

## Path
(e.g. `source/docs/articles/sites/code` or `source/docs/articles/wordpress`)

## Other Considerations
 - Create a walkme segmented for Drupal 8 testing org site dashboards to link to this doc and possibly walk through the process.

### PR DESCRIPTION
**Pull request by bmackinney**
Closes #940

### CODE REVIEW

File: source/docs/articles/drupal/8/moving-out-of-testing.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+1. [Update Drupal to the latest release](/docs/articles/sites/code/applying-upstream-updates).
+1. [Create and download backup archives](/docs/articles/sites/backups).
+2. To use the same URLs for the site, [delete the site](/docs/articles/sites/deleting-a-site/).
+3. [Create a new Drupal 8 site](https://dashboard.pantheon.io/products/drupal8/spinup).
+4. [Import the code, database, and files archives](/docs/articles/sites/migrate/manual-site-import).
+
+
+<div class="alert alert-info" role="alert">
+<h4>Note</h4>
+Sites running versions of Drupal 8 prior to beta 13 do not have a supported upgrade path. If your upstream update fails (step 1), consider creating a new site and migrating your data with another method.</div>
```
Comment (nataliejeremy) [inline source/docs/articles/drupal/8/moving-out-of-testing.md]: @bmackinney Is there another method we can link to here?

File: source/docs/articles/drupal/8/moving-out-of-testing.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+1. [Update Drupal to the latest release](/docs/articles/sites/code/applying-upstream-updates).
+1. [Create and download backup archives](/docs/articles/sites/backups).
+2. To use the same URLs for the site, [delete the site](/docs/articles/sites/deleting-a-site/).
+3. [Create a new Drupal 8 site](https://dashboard.pantheon.io/products/drupal8/spinup).
+4. [Import the code, database, and files archives](/docs/articles/sites/migrate/manual-site-import).
+
+
+<div class="alert alert-info" role="alert">
+<h4>Note</h4>
+Sites running versions of Drupal 8 prior to beta 13 do not have a supported upgrade path. If your upstream update fails (step 1), consider creating a new site and <a href="/docs/articles/sites/code/applying-upstream-updates/#auto-resolve-conflicts">resolving conflicts</a>.</div>
```
Comment (rachelwhitton) [inline source/docs/articles/drupal/8/moving-out-of-testing.md]: Suggest moving this note below first step in ordered list so it's less of an afterthought.

File: source/docs/articles/drupal/8/moving-out-of-testing.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+---
+title: Moving Drupal 8 Sites Out of Testing
+description: Pantheon sites created during the Drupal 8 testing period will be deleted at some point in the future. Follow these instructions to keep your site on the platform.
+---
+The Drupal 8 testing organization owns all Drupal 8 sites created on the platform prior to October 9, 2015. All of these sites will be deleted on <DATE>. To keep your site running on the platform, follow these instructions:
```
Comment (rachelwhitton) [inline source/docs/articles/drupal/8/moving-out-of-testing.md]: @bmackinney do we have a date to include here? Suggest we include it here before adding the doc to the MVP PR.

**Comment by bmackinney** [comment]
closing to re-base pr

**Pull request closed without merging**